packages/@react-spectrum/overlays/stories/Modal.stories.tsx

```diff
@@ -63,15 +63,15 @@ function UnmountingTrigger() {
     <Fragment>
       <DialogTrigger type="popover" isOpen={isPopoverOpen} onOpenChange={setPopoverOpen}>
         <ActionButton>Open popover</ActionButton>
-        <Dialog>
+        <Dialog isDismissable={false}>
```

Comment:
these should already be false? what are these changes for?

Reply:
The responsive Popover hides the footer containing the "Open modal" button when rendered as a Dialog which defaults to `isDismissible={true}`, making it difficult to test if your screen width is too narrow.